tests/fuzzing/oss_fuzz_build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+mv $SRC/id_map_fuzzer.go $SRC/runc/libcontainer/system/
+compile_go_fuzzer ./libcontainer/system FuzzUIDMap id_map_fuzzer linux
```

Comment:
Where is this binary (`compile_go_fuzzer`) coming from?

Reply:
This is defined by OSS-fuzz. Documentation about this has been added inside the build file.